rewrite-java-test/src/test/java/org/openrewrite/java/format/MinimumViableSpacingTest.java

```diff
@@ -65,6 +65,26 @@ class A{public <T> void foo(){}}
         );
     }
 
+    @Test
+    void methodWithThrows() {
+        rewriteRun(
+          spec -> spec.expectedCyclesThatMakeChanges(2),
+          java(
+            """
+              import java.lang.Exception;
+              
+              class A {
+                  public <T> void foo() throws Exception {
+                  }
+              }
+              """,
+            """
+              import java.lang.Exception;class A{public <T> void foo() throws Exception{}}
```

Comment:
Small proposal to minimize the elements in this test to focus purely on `throws`.
```suggestion
                  void foo() throws Exception {
                  }
              }
              """,
            """
              import java.lang.Exception;class A{void foo() throws Exception{}}
```